todoman/cli.py

```diff
@@ -53,14 +53,24 @@ def _validate_list_param(ctx, param=None, name=None):
             raise click.BadParameter(
                 'You must set "default_list" or use -l.'.format(name)
             )
-    for l in ctx.db.lists():
-        if l.name == name:
-            return l
-    else:
-        list_names = [l.name for l in ctx.db.lists()]
-        raise click.BadParameter(
-            "{}. Available lists are: {}".format(name, ', '.join(list_names))
-        )
+    lists = {
+        l.name: l
```

Comment:
`l` is an ambiguous variable name, and flake will complain about it. Please use something like `list_`.